flake.nix

```diff
@@ -295,6 +295,11 @@
       common-gpu-nvidia-sync = import ./common/gpu/nvidia/prime-sync.nix;
       common-gpu-nvidia-nonprime = import ./common/gpu/nvidia;
       common-gpu-nvidia-disable = import ./common/gpu/nvidia/disable.nix;
+      common-gpu-nvidia-ada-lovelace = import ./common/gpu/nvidia/ada-lovelace;
```

Comment:
Does nvidia has any notion of version number for like intel 13th gen?
Assuming that new cards will be supported by the opensource driver, we could than just have a single comparison that would check if the generation number is bigger than X.

Reply:
Unfortunately most other numbers that user facing have been co-opted for marketing purposes and are not reliable to figure out the micro Architecture of the GPU. For the most part they follow the major version of the cards 10xx 20xx and so on. However there are exceptions and those will cause headaches.